src/analysis/retail/warlock/demonology/modules/dragonflight/tier/VaultOfTheIncarnates4Set.tsx

```diff
@@ -0,0 +1,107 @@
+import { formatPercentage } from 'common/format';
+import SPELLS from 'common/SPELLS';
+import Spell from 'common/SPELLS/Spell';
+import { TIERS } from 'game/TIERS';
+import Analyzer, { Options, SELECTED_PLAYER } from 'parser/core/Analyzer';
+import Events from 'parser/core/Events';
+import BoringSpellValueText from 'parser/ui/BoringSpellValueText';
+import Statistic from 'parser/ui/Statistic';
+import STATISTIC_CATEGORY from 'parser/ui/STATISTIC_CATEGORY';
+
+const BLAZING_METEOR: Spell = {
```

Comment:
This should go in `src/common/SPELLS/warlock.ts`

Reply:
Moved it.
I liked the idea of the spell being defined near the only place that uses it though.